README.md

```diff
@@ -3,7 +3,7 @@
 This repository contains a boilerplate static site generator setup for creating CNCF documentation projects. We strongly recommend using this setup (it helps us help you and your project!), but none of the technologies in the stack are strictly required.
 
 The starter uses the following:
-* **[Hugo](https://gohugo.io/)** as a static site generator
+* **[Hugo](This requires Hugo 9.73+ extended version)(https://gohugo.io/)** as a static site generator
```

Comment:
For markdown links, the whole text to be linked needs to be within the `[]` brakets

```suggestion
* **[Hugo (This requires Hugo 9.73+ extended version)](https://gohugo.io/)** as a static site generator
```